cpp/src/text/subword/bpe_tokenizer.cu

```diff
@@ -155,7 +152,7 @@ struct byte_pair_encoding_fn {
     // Edge case check.
     // Empirically found only two merge pair strings that were greater than 70 bytes
     // and they both looked like ignorable errors. Double check this analysis with Vibhu.
```

Comment:
Is "Double check this with Vibhu" an action item needed here, or is this comment outdated?